sdks/java/core/src/main/java/org/apache/beam/sdk/util/ReshuffleTrigger.java

```diff
@@ -32,7 +34,8 @@
  */
 public class ReshuffleTrigger<W extends BoundedWindow> extends Trigger {
 
-  ReshuffleTrigger() {
+  @VisibleForTesting
```

Comment:
Rather than "VisibleForTesting", I'd suggest we either add a VisibleForRunner annotation, or use a comment. Otherwise, we'll look at this in N months and say "hmm... tests are no longer using this visibility, let's change it".